server/registry/internal/storage/list.go

```diff
@@ -130,76 +131,80 @@ var apiFields = []filtering.Field{
 	{Name: "labels", Type: filtering.StringMap},
 }
 
-func (c *Client) ListApis(ctx context.Context, parent names.Project, opts PageOptions) (ApiList, error) {
+func (c *Client) ListApis(ctx context.Context, parent names.Project, opts PageOptions) (*rpc.ListApisResponse, error) {
```

Comment:
Was this change necessary to make this optimization? It seems like an orthogonal refactoring.

Reply:
It wasn't necessary to include. At some point while experimenting with possible listing algorithms I decided to make this change because it was convenient for something I was working on.

I can remove it if you want, but I think returning proto models from the storage package is better than returning the database models.

Reply:
I split the tag removal into #508 and dropped the storage interface changes.